src/v2/Apps/ArtistSeries/Components/ArtistSeriesHeader.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react"
+import { Box, Col, Flex, Grid, Row, Sans } from "@artsy/palette"
+import { Media } from "v2/Utils/Responsive"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ArtistSeriesHeader_artistSeries } from "v2/__generated__/ArtistSeriesHeader_artistSeries.graphql"
+
+interface ArtistSeriesHeaderProps {
+  artistSeries: ArtistSeriesHeader_artistSeries
+}
+
+const ArtistSeriesHeader: React.FC<ArtistSeriesHeaderProps> = props => {
+  return (
+    <>
+      <Media greaterThanOrEqual="sm">
+        <ArtistSeriesHeaderLarge {...props} />
+      </Media>
+      <Media lessThan="sm">
+        <ArtistSeriesHeaderSmall {...props} />
+      </Media>
+    </>
+  )
+}
+
+const ArtistSeriesHeaderLarge: React.FC<ArtistSeriesHeaderProps> = props => {
+  const {
+    artistSeries: { title, description },
+  } = props
+  return (
+    <Box>
+      <Grid>
+        <Row>
+          <Col sm={6}>
+            <Flex
+              height="100%"
+              flexDirection="column"
+              justifyContent="space-between"
+            >
+              <Sans size="8" element="h1" unstable_trackIn>
+                {title}
+              </Sans>
+              <Sans lineHeight={1.5} size="3t" pr={1}>
+                {description}
+              </Sans>
+            </Flex>
+          </Col>
+          <Col sm={6}>
+            <Box
+              // FIXME: Use Image
+              bg="black10"
+              width={1}
+              height={400}
+            ></Box>
+          </Col>
+        </Row>
+      </Grid>
+    </Box>
+  )
+}
+
+const ArtistSeriesHeaderSmall: React.FC<ArtistSeriesHeaderProps> = props => {
+  const {
+    artistSeries: { title, description },
```

Comment:
#minor might be nice to keep this generic-y and pass in `title` and `description` (so the component doesn't need any `artistSeries` naming/knowledge. Then it may make it easier to decide if it's a component that itself should be moved to Palette (or if there's an existing similar component).

Reply:
Which component do you mean? I.e. do you think the two `ArtistSeriesHeader` components could be individual Palette components?

Reply:
IMO these components seem simple enough constructions out of our palette building blocks that they don't need to be palette components in themselves (but that could certainly change at some point if we end up repeating ourselves in many places).

Reply:
Yea, they _might_ have a future as being more generic (pure speculation on my part, not at all sure), so rather than taking in a 'custom' `artistSeries` prop (that it has to destruct for the `title` and `description`), it might be easier to skip that and pass them in directly. More of a general comment!

Reply:
Cool -- I'll leave it as-is for now since the scope for this type of component so far os just for artist series, but we can return to the idea of Palette-ization as the work continues.